package.json

```diff
@@ -3,10 +3,20 @@
   "description": "initials for names",
   "main": "initials.js",
```

Comment:
as we have `index.js` now, remove this line, otherwise `require('initials')` will fail